src/NCronJob/Scheduler/JobQueueManager.cs

```diff
@@ -47,7 +47,15 @@ public void RemoveQueue(string queueName)
         {
             if (jobQueues.TryRemove(queueName, out var jobQueue))
             {
-                // TODO: Should we first cancel the potentially existing JobRuns here?
+                foreach (var job in jobQueue)
+                {
+                    if (!job.IsCancellable)
```

Comment:
nitpick: Could be done in a `Where(j => j.IsCancellable)`

Reply:
Fixed